packages/crawler/src/page-operations/accessibility-scan-operation.ts

```diff
@@ -9,14 +9,18 @@ import { BlobStore } from '../storage/store-types';
 export class AccessibilityScanOperation {
     constructor(@inject(PageScanner) private readonly scanner: PageScanner) {}
 
-    public async run(page: Page, id: string, keyValueStore: BlobStore): Promise<void> {
+    public async run(page: Page, id: string, keyValueStore: BlobStore): Promise<number> {
```

Comment:
Logically we may need some structure to return complete result instead of just single number. Could be like
`{
passed:number,
failed:number;
unscannable: number;
}`

Reply:
For now we need only the issue count